### ISSUE TITLE
[a11y] Walkthrough with Accessible View should not inject Step N, or walkthrough UX should be updated

### ISSUE THREAD

**Issue by pwang347** [microsoft/vscode#230334]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.94.0-insider
- OS Version: Mac Sonoma 14.6.1

Steps to Reproduce:

1. Open a walkthrough that has "Step N" already in the step title
2. Run the command: `Open Accessible View`

<img width="654" alt="image" src="https://github.com/user-attachments/assets/4e64a62c-479d-4f94-84c2-a4e7edfc0cc4">

Should "Step 1" be included in the accessible view (or alternatively, should Step N be automatically included by VS Code into walkthroughs)? The walkthrough steps UX seem to imply an unordered list using the radio buttons, so we added our own step number label but this leads to a duplicated string in the accessible view.

Thanks!
